viewer/viewer.js

```diff
@@ -283,13 +285,34 @@ if (Config.get("passwordSecret")) {
           userCleanup(suser._source);
           req.user = suser._source;
           return next();
+        };
+
+        Db.getUserCache(userName, function(err, suser) {
+          if (internals.userAutoCreateTmpl === undefined) {
+             return ucb(err, suser);
+          } else if ((err && (err+"").includes("Not Found")) ||
```

Comment:
`err.toString().includes(` is considered cleaner even though the `err+""` will work

Reply:
Updated.  I wasn't sure if err was always an object that supports toString(), but I'll take your word for it since this is the first time I've written JavaScript since my HTML4.0 class in 1998.